portfolio/src/main/webapp/script.js

```diff
@@ -42,13 +42,17 @@ function topFunction() {
 // https://www.quora.com/How-can-I-make-a-comment-box-in-HTML
 // This code gets comment data from the /data backend path and updates the #display-comments element
 // with comment data.
-fetch('/data').then(response => response.json()).then((comments) => {
+fetch('/data').then(response => response.json()).then((data) => {
     var displayComments = document.getElementById('display-comments');
+    var comment;
+    var email;
```

Comment:
These variables are not needed (and can confuse the readers), it'll be  much simpler if you do the following:

`displayComments.innerHTML += "" + data.email[i] + ": "+ data.comments[i] + "<br/>";`

Reply:
fixed